website/react-magma-docs/src/pages/api/stepper.mdx

```diff
@@ -759,6 +759,279 @@ export function Example() {
+            <Step key={1} label="Second Item" secondaryLabel="Description Two">
+              Item Content Two
+            </Step>
+            <Step key={2} label="Third Item" secondaryLabel="Description Three">
+              Item Content Three
+            </Step>
+            <Step key={3} label="Fourth Item" secondaryLabel="Description Four">
+              Item Content Four
+            </Step>
+            <Step key={4} label="Fifth Item" secondaryLabel="Description Five">
+              Item Content Five
+            </Step>
+          </Stepper>
+        </DropdownContent>
+      </Dropdown>
+      <Dropdown>
+        <DropdownButton>Horizontal Stepper</DropdownButton>
+        <DropdownContent
+          style={{
+            whiteSpace: 'normal',
+            maxHeight: 'fit-content',
+            padding: '1em',
+          }}
+        >
+          <Stepper currentStep={2} ariaLabel="progress">
+            <Step key={0} label="First Item" secondaryLabel="Description One">
+              Item Content One
+            </Step>
+            <Step key={1} label="Second Item" secondaryLabel="Description Two">
+              Item Content Two
+            </Step>
+            <Step key={2} label="Third Item" secondaryLabel="Description Three">
+              Item Content Three
+            </Step>
+            <Step key={3} label="Fourth Item" secondaryLabel="Description Four">
+              Item Content Four
+            </Step>
+            <Step key={4} label="Fifth Item" secondaryLabel="Description Five">
+              Item Content Five
+            </Step>
+          </Stepper>
+        </DropdownContent>
+      </Dropdown>
+    </div>
+  );
+}
+```
+
+## Breakpoint Orientation
+
+For responsive layouts, the `breakpoint` prop can be used in combination with the `breakpointOrientation` prop to change the `Stepper`’s orientation based on screen width. The `breakpoint` prop sets a maximum screen width, and `breakpointOrientation` defines the desired orientation (horizontal or vertical) when the screen width is below the specified breakpoint. By default, the `Stepper` is displayed horizontally if no responsive props are used. It is recommended to use `breakpointOrientation` with `ResponsibleStepperContainer` for optimal behavior.
+
+```typescript
+import React from 'react';
+import {
+  Container,
+  Stepper,
+  StepperOrientation,
+  Step,
+  Button,
+  ButtonGroup,
+  Flex,
+  FlexBehavior,
+  ResponsibleStepperContainer,
+} from 'react-magma-dom';
+
+export function Example() {
+  const [currentStep, setCurrentStep] = React.useState(0);
+
+  const handleOnNext = () => {
+    if (currentStep !== 4) {
+      setCurrentStep(currentStep + 1);
+    }
+  };
+  const handleOnPrevious = () => {
+    if (currentStep !== 0) {
+      setCurrentStep(currentStep - 1);
+    }
+  };
+  const handleFinish = () => {
+    if (currentStep === 4) {
+      setCurrentStep(0);
+    }
+  };
+
+  return (
+    <ResponsibleStepperContainer
```

Comment:
I wonder if we can make the API a little more straight forward to use. Maybe something like this? What do you think?

```
<ResponsiveStepperContainer
  breakpoint={1200}
  breakpointOrientation={StepperOrientation.vertical}
  breakpointLayout={StepperLayout.hideLabels}
  currentStep={2}
  ariaLabel="progress"
  orientation={StepperOrientation.vertical}
  steps={
    <>
      <Step key={0} label="First Item" secondaryLabel="Description One">
        {" "}
        Item Content One
      </Step>
      <Step key={1} label="Second Item" secondaryLabel="Description Two">
        {" "}
        Item Content Two
      </Step>
      <Step key={2} label="Third Item" secondaryLabel="Description Three">
        {" "}
        Item Content Three
      </Step>
      <Step key={3} label="Fourth Item" secondaryLabel="Description Four">
        {" "}
        Item Content Four
      </Step>
      <Step key={4} label="Fifth Item" secondaryLabel="Description Five">
        {" "}
        Item Content Five
      </Step>
    </>
  }
>
  {/* Content */}
  <Container
    style={{
      background: "#F5F5F5",
      borderRadius: "6px",
      padding: "20px",
      width: "100%",
      flex: 10,
    }}
  >
    {currentStep === 0 && <div>Step Content One</div>}
    {currentStep === 1 && <div>Step Content Two</div>}
    {currentStep === 2 && <div>Step Content Three</div>}
    {currentStep === 3 && <div>Step Content Four</div>}
    {currentStep === 4 && <div>Steps Completed</div>}
  </Container>

  <Flex
    behavior={FlexBehavior.item}
    style={{ padding: "20px 0", alignSelf: "flex-end" }}
  >
    <ButtonGroup>
      <Button disabled={currentStep === 0} onClick={handleOnPrevious}>
        Previous
      </Button>
      <Button onClick={currentStep >= 4 ? handleFinish : handleOnNext}>
        {currentStep >= 4 ? "Finish" : "Next"}
      </Button>
    </ButtonGroup>
  </Flex>
</ResponsiveStepperContainer>
```